apps/meteor/tests/end-to-end/api/06-outgoing-integrations.js

```diff
@@ -16,22 +16,24 @@ describe('[Outgoing Integrations]', function () {
 	let userCredentials;
 
 	before((done) => getCredentials(done));
-
-	before((done) => {
-		updatePermission('manage-incoming-integrations', [])
-			.then(() => updatePermission('manage-own-incoming-integrations', []))
-			.then(() => updatePermission('manage-own-outgoing-integrations', []))
-			.then(() => updatePermission('manage-outgoing-integrations', []))
-			.then(done);
+	before(async () => {
+		await Promise.all([
+			updatePermission('manage-incoming-integrations', []),
+			updatePermission('manage-own-incoming-integrations', []),
+			updatePermission('manage-own-outgoing-integrations', []),
+			updatePermission('manage-outgoing-integrations', []),
+		]);
 	});
 
-	after((done) => {
-		updatePermission('manage-incoming-integrations', ['admin'])
-			.then(() => updatePermission('manage-own-incoming-integrations', ['admin']))
-			.then(() => updatePermission('manage-own-outgoing-integrations', ['admin']))
-			.then(() => updatePermission('manage-outgoing-integrations', ['admin']))
-			.then(done);
-	});
+	after(async () =>
+		Promise.all([
+			updatePermission('manage-incoming-integrations', ['admin']),
+			updatePermission('manage-own-incoming-integrations', ['admin']),
+			updatePermission('manage-own-outgoing-integrations', ['admin']),
+			updatePermission('manage-outgoing-integrations', ['admin']),
+			deleteUser(user),
```

Comment:
should this user be created in the above `before` section as well? or should it be deleted closer to where it was created? 😬

Reply:
It should be created in a proper `before`. Thanks for noticing that 👍